gp-includes/schema.php

```diff
@@ -82,16 +82,16 @@ function gp_schema_get() {
 	 *
 	 */
 	$gp_schema['originals'] = "CREATE TABLE $wpdb->gp_originals (
-		id INT(10) NOT NULL auto_increment,
-		project_id INT(10) DEFAULT NULL,
-		context VARCHAR(255) DEFAULT NULL,
-		singular TEXT NOT NULL,
-		plural TEXT DEFAULT NULL,
-		`references` TEXT DEFAULT NULL,
-		comment TEXT DEFAULT NULL,
-		status VARCHAR(255) NOT NULL DEFAULT '+active',
-		priority TINYINT NOT NULL DEFAULT 0,
-		date_added DATETIME DEFAULT NULL,
+		id int(10) NOT NULL auto_increment,
+		project_id int(10) DEFAULT NULL,
+		context varchar(255) DEFAULT NULL,
+		singular text NOT NULL,
+		plural text DEFAULT NULL,
+		`references` text DEFAULT NULL,
+		comment text DEFAULT NULL,
+		status varchar(255) NOT NULL DEFAULT '+active',
+		priority tinyint(4) NOT NULL DEFAULT 0,
```

Comment:
See discussion here: https://wordpress.slack.com/archives/glotpress/p1465319555000063

Size is required for `dbDelta()`.